python/ic_azure/azure_discover_resources.py

```diff
@@ -23,15 +24,14 @@ def find_resources(self):
         # List resources and tags using client
         for resource in self._client.resources.list():
 
-            # Split resource ID
-            id_splitted = resource.id.split("/")
+            type_match = re.search('providers(\/[^\/]+){1,4}', resource.id).group(0)
```

Comment:
Why would you use RegEx in these? I mean, it's fine but when you can split using "/" it's way more readable that way...

Reply:
Because we want to take values for the macros from certain positions in the path string. Paths vary and i.e. resource type is not always in the same position (not always the 6th and 7th in the splitted list) . Regex just came to my mind first.